bpf/lib/nodeport.h

```diff
@@ -1113,6 +1117,12 @@ static __always_inline int nodeport_nat_ipv4_fwd(struct __ctx_buff *ctx)
 	if (ret == NAT_PUNT_TO_STACK)
 		ret = CTX_ACT_OK;
 
+	/* If multiple netdevs process an outgoing packet, then this packets will
+	 * be handled multiple times by the "to-netdev" section. This can lead
+	 * to multiple SNATs. To prevent from that, set the SNAT done flag.
+	 */
+	ctx_snat_done_set(ctx);
```

Comment:
Could you s/ctx_nat_done_set/bpf_mark_snat_done/?

Reply:
Thanks, amended!